lib/ansible/modules/cloud/amazon/kinesis_stream.py

```diff
@@ -1069,7 +1075,7 @@ def main():
             )
         )
     except botocore.exceptions.ClientError as e:
-        err_msg = 'Boto3 Client Error - {0}'.format(str(e.msg))
+        err_msg = 'Boto3 Client Error - {0}'.format(to_native(e.msg))
         module.fail_json(
             success=False, changed=False, result={}, msg=err_msg
```

Comment:
What if, at line 931 (or less likely 866) StreamStatus is missing.  Shouldn't we use get in those cases too?